### ISSUE TITLE
Feature request: All col-groups for medium and small layouts

### ISSUE THREAD

**Issue by sebastianhelbig** [daneden/Toast#25]
As of now it is restricted to -of-2, -of-3 and -of-4. I would like to have all col-groups available.

**Comment by Swimburger** [comment]
Agreed, it would be great to be able to define the col-groups for other breakpoints.
